packages/mobile/src/screens/notifications-screen/Notifications/MilestoneNotification.tsx

```diff
@@ -91,8 +95,9 @@ type MilestoneNotificationProps = {
 export const MilestoneNotification = (props: MilestoneNotificationProps) => {
   const { notification } = props
   const { achievement } = notification
-  const entity = useSelector((state) =>
-    getNotificationEntity(state, notification)
+  const entity = useProxySelector(
+    (state) => getNotificationEntity(state, notification),
+    [notification]
```

Comment:
so it is the case that "notification" object is stable?

Reply:
I believe so, this is how the other notif types are doing it